ENIGMAsystem/SHELL/Audio_Systems/OpenAL/Makefile

```diff
@@ -1,3 +1,7 @@
+ifndef PKG-CONFIG
```

Comment:
I think this belongs in https://github.com/enigma-dev/enigma-dev/blob/master/Config.mk and the SDL one can be deleted but needs testing on CI